src/subscription_manager/migrate/migrate.py

```diff
@@ -611,7 +611,7 @@ def disable_yum_rhn_plugin(self):
         /etc/yum/pluginconf.d/rhnplugin.conf
         Can thrown IOError exception.
         """
-        log.info("Disabling rhnplugin.conf")
+        log.debug("Disabling rhnplugin.conf")
```

Comment:
When plugin is disabled, then it is change of state. I would keep `info` here too.